.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+### Are you requesting a feature or reporting a bug?
+
+
+
+### What is the current behavior?
+
+
+
+### How would you reproduce this behavior (if this is a bug)?
+
+
+
+### What is the expected behavior?
```

Comment:
Don't we have it?

https://github.com/DevExpress/testcafe/pull/868/files#diff-1a1c3dd142f76a5fad803a0c52839881R5